locales/de/validation-inline.php

```diff
@@ -96,11 +96,11 @@
     'numeric'              => 'Das muss eine Zahl sein.',
     'password'             => 'Das Passwort ist falsch.',
     'present'              => 'Dieses Feld muss vorhanden sein.',
-    'prohibited'           => 'Dieses Feld ist verboten.',
-    'prohibited_if'        => 'Dieses Feld ist verboten, wenn :other gleich :value ist.',
-    'prohibited_unless'    => 'Dieses Feld ist verboten, wenn :other nicht in :values enthalten ist.',
+    'prohibited'           => 'Dieses Feld ist gesperrt.',
```

Comment:
https://dict.leo.org/englisch-deutsch/prohibit

"gesperrt" has a different meaning. An alternative would be "untersagt", whereby "verboten" is more correct.

Reply:
This one was the only one where I could find the PR that introduced it:

> The basic idea of "prohibited" validation rules is that a given field should be prohibited from having data if another field is present or if a field should be allowed in a request at all.
[Source](https://laravel-news.com/laravel-prohibited-validation-rules)

While I'm happy to discuss better wording, I do not think "verboten" is the best explanation for what's going on here. I usually look up the words for translation, IMO it should better match the meaning rather than a hard translation.

Reply:
You are right. "verboten" was a bad translation. What do you think of "unzulässig"?